### DISCUSSION TITLE
Button "+" (for adding new elements) not working/responding

### DISCUSSION THREAD

**Discussion by lubiku35** [GitbookIO/community#338] [0. General bugs and questions]
hello I came across several bugs today which are continuously happening to me all day, if I click on the "+" button nothing happens, the page does not respond, when I use the character "/" I have no prediction, sometimes I can't move by arrows to navigate in the text, maybe I just missed that something is being updated, thank you for the info

**Comment by lubiku35** [comment]
Update: The problem is in type of browser, defaultly brave brovser is blocking this interactions, if i switch to another browser (chromium) it's working as normal.

**Reply by aikoii** [reply]
Thanks for the heads up @lubiku35! Also, if you switch Shields down everything should be working as expected.

<img width="340" alt="CleanShot 2023-09-20 at 11 58 01@2x" src="https://github.com/GitbookIO/community/assets/106068994/3a11fcd6-284f-4924-b959-ac34e3f43717">

**Reply by aikoii** [reply]
Quick update, this was fixed 🎉. Our app is now good to go with Shields up.
